site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/converter/SiteConverter.java

```diff
@@ -17,10 +21,27 @@ public Site convert(final Organization org) {
                 org.getName(),
                 (org.getAlias().isEmpty()) ? null : org.getAlias().get(0).getValueAsString(),
                 findParentSiteId(org),
-                org.getImplicitRules()
+                org.getImplicitRules(),
+                findAddress(org)
         );
     }
 
+    private Address findAddress(Organization org) {
+        if (org.hasAddress()) {
+            List<StringType> line = org.getAddress().get(0).getLine();
+            return new Address(!line.isEmpty() ? line.get(0).getValue() : "",
+                    !line.isEmpty() && line.size() > 1 ? line.get(1).getValue() : "",
+                    !line.isEmpty() && line.size() > 2 ? line.get(2).getValue() : "",
+                    !line.isEmpty() && line.size() > 3 ? line.get(3).getValue() : "",
+                    !line.isEmpty() && line.size() > 4 ? line.get(4).getValue() : "",
+                    org.getAddress().get(0).getCity() != null ? org.getAddress().get(0).getCity() : "",
+                    org.getAddress().get(0).getCountry() != null ? org.getAddress().get(0).getCountry() : "",
+                    org.getAddress().get(0).getPostalCode() != null ? org.getAddress().get(0).getPostalCode() : "");
+        } else {
+            return null;
+        }
+    }
+
```

Comment:
Ditto above comment for the reverse convertor from FHIR address to model address

Reply:
I have added EntityConverters for SiteAddressConverter (MTS Model) and AddressConverter (FHIR)